src/contour/TerminalSession.cpp

```diff
@@ -826,6 +828,15 @@ void TerminalSession::sendMousePressEvent(Modifiers modifiers,
     inputLog()("Mouse press received: {} {}\n", modifiers, button);
 
     terminal().tick(steady_clock::now());
+    auto const passed =
+        std::chrono::duration_cast<std::chrono::milliseconds>(_terminal.currentTime() - _previousMouseEvent);
+
+    if (passed.count() < 50)
```

Comment:
This feels like we really want that `50` configurable. Because most likely I'd like to set it to `0` and maybe that also depends on the frequency of mouse event reporting on the OS side that Qt is blindingly forwarding (just guessing, but I know I'm having such a gamer mouse that can report in different frequencies)

Reply:
I agree that we should make it configurable, maybe we can postpone it until config refector PR? at the moment this number is quite arbitrary and based only on my machine, but i would not expect actual mouse to show this behaviour since usually there is a tick mechanism there to trigger scroll event